### ISSUE TITLE
Syntax error from jQuery when labels have special characters

### ISSUE THREAD

**Issue by benjamin-fenker-tessella** [LLTommy/OLS-graphview#2]
When accessing the URL: [such as this] where--(http://www.ebi.ac.uk/ols/api/ontologies/ordo/terms/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_950/graph) where one of the edges has a label with special quotes characters such as

`"label" : "Disease-causing germline mutation(s) (gain of function) in",`

the plugin throws an error:

```
Uncaught Error: Syntax error, unrecognized expression: #visible_Disease-causinggermlinemutation(s)(gainoffunction)in
    at Function.ga.error (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:2:12731)
    at ga.tokenize (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:2:18658)
    at ga.select (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:2:21386)
    at Function.ga [as find] (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:2:7364)
    at m.fn.init.find (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:2:23660)
    at m.fn.init (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:2:24231)
    at m (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:2:393)
    at http://localhost:8000/build/ols-graphview.min.js:21:3169
    at Array.forEach (native)
    at http://localhost:8000/build/ols-graphview.min.js:21:3088
ga.error @ jquery.js:1496
ga.tokenize @ jquery.js:2113
ga.select @ jquery.js:2517
ga @ jquery.js:893
find @ jquery.js:2733
m.fn.init @ jquery.js:2850
m @ jquery.js:73
(anonymous) @ ols-graphview.min.js:21
(anonymous) @ ols-graphview.min.js:21
E @ ols-graphview.min.js:21
(anonymous) @ ols-graphview.min.js:20
j @ jquery.js:3148
fireWith @ jquery.js:3260
x @ jquery.js:9314
b @ jquery.js:9718
```

It looks like jQuery cannot parse selectors with `)` in them, which is the underlying cause. Do you have any suggestions on how to dodge this issue?

Thanks!

**Comment by LLTommy** [comment]
> Do you have any suggestions on how to dodge this issue?

I updated the code on github as well as on npm, should fix this specific problem. I will make sure the graphview in OLS is updated as well.

**Comment by benjamin-fenker-tessella** [comment]
Thanks, it works great!
